kaizen/frontend/ui/src/index.css

```
@@ -0,0 +1,870 @@
+  --success: #10B981;
+  --danger: #EF4444;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)